src/util/mod.rs

```diff
@@ -5,7 +5,7 @@ use std::collections::HashMap;
 use serde::Serialize;
 use std::marker::PhantomData;
 
-pub(crate) fn build_map<'key, 'value, const N: usize>(
+pub fn build_map<'key, 'value, const N: usize>(
```

Comment:
Will this change the visibility from `crate` public to globally public?

Reply:
Because the entire module is already public to the crate only. No need to do that again.